app/public/cantusdata/management/commands/import_data.py

```diff
@@ -89,8 +89,8 @@ def import_manuscript_data(self, **options):
                 # Getting the fields from the scraper
                 id = source.get("id")
                 name = source.get("name")
-                cantus_url = f"{cdb_base_url}source/{id}"
-                csv_export_url = f"{cdb_base_url}sites/default/files/csv/{id}.csv"
+                cantus_url = f"{cdb_base_url}/source/{id}"
```

Comment:
maybe this variable could be `cantusdb_url`, to avoid confusion with other notable Cantuses :P